README.md

```diff
@@ -179,11 +178,13 @@ Total mutants killed: 203
 ├────────────────────────────────────────────────┼────────────────┼────────────────┼────────────┤
 │ sources/Sum.move::Sum::sum                     │ 4              │ 4              │ 100.00%    │
 ╰────────────────────────────────────────────────┴────────────────┴────────────────┴────────────╯
+Total mutants tested: 229
+Total mutants killed: 203
 ```
 
 The sample `report.txt` generated for the above command contains useful info that can be paired with the `display-report` option:
 ```bash
-$ move-mutation-test display-report -p report.txt --modules Sum
+$ move-mutation-test display-report coverage --path-to-report report.txt --modules Sum
```

Comment:
As mentioned elsewhere, the `spec` in the code for `fun sum` is an unnecessary distraction here, we should remove it for `move-mutation-test` related descriptions, like here.